lib/pure/asyncdispatch.nim

```diff
@@ -1009,7 +1009,7 @@ else:
     processTimers(p)
 
   proc connect*(socket: AsyncFD, address: string, port: Port,
-    af = AF_INET): Future[void] =
+    af_unused = AF_INET): Future[void] =
```

Comment:
Yeah, the best would be to test and update the windows version as well, but I don't have the access to Windows platform to test, so I left it this way for now...